pkg/auth/providers/oidc/oidc_actions.go

```diff
@@ -112,3 +117,15 @@ func (o *OpenIDCProvider) TestAndApply(actionName string, action *types.Action,
 
 	return o.TokenMGR.CreateTokenAndSetCookie(user.Name, userPrincipal, groupPrincipals, providerToken, 0, "Token via OIDC Configuration", request, userExtraInfo)
 }
+
+func validScopes(input string) bool {
+	values := strings.Fields(input)
+
+	// Iterate through the values and check if any of them is "openid"
+	for _, value := range values {
+		if value == "openid" {
+			return true
+		}
+	}
+	return false
```

Comment:
You could simplify this to `return slices.Contains(values, "openid")`

But no tests for this?

Reply:
why do I still forget about slices.Contains? :)
Added check for commas in addition to this.
Test added as well.